core-designsystem/src/main/java/com/google/samples/apps/nowinandroid/core/designsystem/component/ToggleButton.kt

```diff
@@ -72,7 +72,9 @@ fun NiaToggleButton(
         onClick = { onCheckedChange(!checked) },
         modifier = modifier
             .size(size)
-            .toggleable(value = checked, enabled = enabled, role = Role.Button, onValueChange = {})
+            .toggleable(value = checked, enabled = enabled, role = Role.Button, onValueChange = {
+                onCheckedChange(!checked)
```

Comment:
Nit: Should this use the parameter of `onValueChange` instead of `checked`?

Probably doesn't really matter, they should do the same thing.

Reply:
yeah, i think it's essentially the same. will keep it as is!